lib/src/ui/common/theme/theme.dart

```diff
@@ -1,52 +1,73 @@
 import 'package:flutter/material.dart';
 import 'package:google_fonts/google_fonts.dart';
 
-final ThemeData newsTheme = _buildNewsTheme();
+final ThemeData lightNewsTheme = _buildLightNewsTheme();
+final ThemeData darkNewsTheme = _buildDarkNewsTheme();
 
-ThemeData _buildNewsTheme() {
+class _Constants {
+  static const buttonColor = Color(0xFFE0E0E0);
+}
+
+ThemeData _buildLightNewsTheme() {
   final base = ThemeData.light();
 
   return base.copyWith(
+    backgroundColor: Colors.white,
     colorScheme: const ColorScheme.light(),
-    textTheme: _buildNewsTextTheme(base.textTheme),
+    buttonColor: _Constants.buttonColor,
+    accentColor: Colors.black54,
+    textTheme: _buildNewsTextTheme(base.textTheme, Colors.black),
```

Comment:
co w sytuacji gdybyśmy mieli więcej kolorów tekstów w aplikacji i przechodziłyby jeden w drugi? Przy obecnym podejściu każdy kolor trzeba rozpatrywać osobno

Reply:
hmm... chyba nie do końca rozumiem, jakby jeden kolor przechodził w drugi to by trzeba było to uwzględnić jakoś, ale tak nie jest - z tego co widzę to chyba każdy textStyle tak czy siak trzeba rozpatrywać indywidualnie - chyba, że czegoś nie zauważyłem. Masz jakieś konkretne podejście na myśli?

Reply:
moja pierwsza myśl była taka, że może to być problematyczne przy wielu kolorach tekstów, ale racja nie widzę innego wyjścia jak przekazywanie kolorów dla stylów osobno